src/components/Pf/PfInfoCard.tsx

```diff
@@ -24,18 +24,16 @@ class PfInfoCard extends React.Component<CardProps> {
           <h2 className="card-pf-title">
             <Icon type={this.props.iconType} name={this.props.iconName} style={{ marginRight: '10px' }} />
             {this.props.title}
-            {this.props.istio ? (
-              this.props.showOnGraphLink && (
-                <>
-                  {' '}
-                  (<Link to={this.props.showOnGraphLink}>Show on graph</Link>)
-                </>
-              )
-            ) : (
+            {this.props.title && this.props.istio !== undefined && !this.props.istio && (
               <span style={{ marginLeft: '10px' }}>
                 <MissingSidecar />
               </span>
             )}
+            {this.props.title && this.props.showOnGraphLink && (
```

Comment:
Something changed, not sure why.
Previous behavior was to hide "Show on graph" if sidecar is missing. Now, it is always present.

Reply:
Yes, I changed that on purpose because I didn't see a reason to not show it on the graph if it was lacking a sidecar.  The graph also shows nodes lacking a sidecar,  especially when the "show unused" option is selected, which is true when this link is generated, right?  Am I missing something?

Reply:
If it was intentional, it is OK. I have no strong opinion.